### PR TITLE
create another module to use another ipinfo api

### PR DESCRIPTION
**Pull request by Fengyalv**
The original ip API has a rate limit for 1000 per day, which is a little bit easy to reach.

This module use an API from `http://ip-api.com`, which has a higher rate limit for 120 per minute.

This API also provide more informations about ip such as `AS`, `timezone`.

### CODE REVIEW

File: ipinfohigherlimit/widget.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+// Refresh refresh the module
+func (widget *Widget) Refresh() {
+	if widget.Disabled() {
+		return
+	}
+
+	widget.UpdateRefreshedAt()
+	widget.ipinfo()
+	widget.View.Clear()
```
Comment (jeangovil) [inline ipinfohigherlimit/widget.go]: Don't need this when use `widget.View.SetText()` to print text.

File: ipinfohigherlimit/widget.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	widget.View.SetWrap(false)
+
+	widget.config()
+
+	return &widget
+}
+
+// Refresh refresh the module
+func (widget *Widget) Refresh() {
+	if widget.Disabled() {
```
Comment (jeangovil) [inline ipinfohigherlimit/widget.go]:
```
if widget.Disabled() {
	return
}
```
It's not necessary any more.

File: ipinfohigherlimit/widget.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+		return
+	}
+	req.Header.Set("User-Agent", "curl")
+	response, err := client.Do(req)
+	if err != nil {
+		widget.result = fmt.Sprintf("%s", err.Error())
+		return
+	}
+	defer response.Body.Close()
+	if err != nil {
```
Comment (jeangovil) [inline ipinfohigherlimit/widget.go]:
```
if err != nil {
	widget.result = fmt.Sprintf("%s", err.Error())
	return
}
```
Sorry I can't see the reason for checking the same `err` two times(lines 73-76 & 78-81)

Comment (Fengyalv) [reply ipinfohigherlimit/widget.go]: Sorry, this part is from the original ipinfo. So I didn't change it. I will correct it.

**Comment by senorprogrammer** [comment]
I like the module, I don't like the name. `ipapi` seems like a reasonable name for it, given the service it uses.

**Comment by senorprogrammer** [comment]
Thanks!

**Pull request merged**